src/data_vis.ts

```diff
@@ -148,8 +148,7 @@ class DataVisModel {
     groups: countryGroups,
     humanNames: countryDict,
     name: "Country",
-    options: countries,
-    selected: countryGroups["pine"],
+    options: countries
```

Comment:
Default behaviour of this class is to use `options` as the initial selection if no explicit `selected` passed, so this will simply select all countries.